SmellDetectorTests/Unit/SmellDetectorServiceTest.cs

```diff
@@ -6,12 +6,48 @@
 using System.Linq;
 using Xunit;
 using Shouldly;
+using SmellDetector.Communication;
+using SmellDetectorTests.DataFactory;
 using System.Collections.Generic;
 
 namespace SmellDetectorTests.Unit
 {
     public class SmellDetectorServiceTest
     {
+
+        [Fact]
+        public void Generate_Smell_Detection_Report_For_LongMethod_And_Long_Parameter_List_Issues()
+        {
+            DetectionService detectionService = new DetectionService();
```

Comment:
There are (at least) 3 major issues with these tests. Think about what they are, so that we can discuss them on Friday.